packages/blockchain-wallet-v4-frontend/src/modals/DebitCard/OrderCard/ResidentialAddress/ResidentialAddress.tsx

```diff
@@ -0,0 +1,361 @@
+import React, { useEffect } from 'react'
+import { FormattedMessage } from 'react-intl'
+import { useDispatch, useSelector } from 'react-redux'
+import { validate } from 'postal-codes-js'
+// @ts-ignore
+import postalCodes from 'postal-codes-js/generated/postal-codes-alpha2'
+import { path } from 'ramda'
+import { Field, InjectedFormProps, reduxForm } from 'redux-form'
+import styled from 'styled-components'
+
+import { CountryScope } from '@core/types'
+import { BlockchainLoader, Button, HeartbeatLoader, Text } from 'blockchain-info-components'
+import FlyoutContainer from 'components/Flyout/Container'
+import FlyoutContent from 'components/Flyout/Content'
+import FlyoutFooter from 'components/Flyout/Footer'
+import FlyoutHeader from 'components/Flyout/Header'
```

Comment:
Could do these all on one line if you want `import { FlyoutContainer, FlyoutContent, FlyoutFooter, FlyoutHeader } from 'components/Flyout/Layout'`